SparkPost/2web2ui
# FE-1139 - Batch #4 of UX and Visual Enhancements Across Features

[FE-1139](https://jira.int.messagesystems.com/browse/FE-1139)

### What Changed
- Resolves issues outlined in the ticket
- Updates unit tests and removes snapshots where possible
- **Note:** Failing integration test that is a false positive - addressed as a part of #1723 

### How To Test
- Within the OG and Hibana themes, go to `account/settings` - edit `SingleSignOnPanel.js` to render the "Single sign-on is not available while two-factor authentication is required on this account." paragraph - and verify spacing renders correctly
- Within the Hibana theme, open the popover in the top right of the screen (by clicking on the orange circle in the top right). Verify all icons are removed except for the external icon for the API docs.
- Within the Hibana them, verify the padding at the bottom of the screen is much larger - this is easier to see on the `/reports/message-events` page than on the dashboard
- In the OG theme, go to `/templates` and in the "Recent Activity" section, open the popover and click on "Duplicate Template" - verify the modal is properly covered by the modal overlay

### To Do
